AbhinavGoel9/openpilot
# feat: Add sample feature template and test PR review bot

### Description
This pull request is created to test the functionality of the PR review bot.

### Changes Made
- Added a sample feature template file: `sample_feature.md`

### Testing Steps
1. Confirm that the PR review bot correctly identifies the new feature template.
2. Check if the `bot-review` label is added as expected.
3. Verify that the draft status is updated according to the logic in the Python script.
